Feature/plat 277/add a pgedge distributed example to postgres images (#1)
pgEdge/postgres-images: README.md

### docker compose

> This repository includes two Compose examples you can try out:

Reviewer: ```suggestion
This repository includes two Docker Compose examples you can try out:
```

> - Enterprise Example

Reviewer: ```suggestion
#### Enterprise Example
```

> Runs a single Postgres instance using the standard image and initializes extensions.

Reviewer: ```suggestion
Runs a single Postgres instance using the standard image and initializes common extensions.
```

> 

Reviewer: Rather than including instructions here, we should link to a short README in the enterprise directory.

- `cd examples/compose/enterprise`
- `docker compose up`

> This example utilizes the standard image and handles initializing and creating extensions.

Reviewer: ```suggestion

```

I'd remove this line with the above update

> - Distributed Example

Reviewer: ```suggestion
#### Distributed Example
```

> Runs two pgEdge Postgres nodes (n1 / n2) with Spock logical replication preconfigured.

Reviewer: ```suggestion
Runs two Postgres instances as pgEdge nodes (n1 / n2) with Spock logical replication pre-configured.
```

This example demonstrates bi-directional replication and includes instructions for testing with the Northwind dataset.

> Once Postgres initialization completes, you can connect to the database in a separate shell:

Reviewer: I would suggest removing these extra lines and linking to the example directory which has the other README
